docs/src/examples.md

```diff
@@ -41,7 +41,59 @@ julia> round.(predict(ols), digits=5)
  6.83333
 ```
 
-.
+## Categorical variables/dummy coding
```

Comment:
This isn't really an example of a model, so better put this section elsewhere.

Reply:
I put it there as an example of use of the library. I could instead put it in the manual, but a full example seems like an odd thing to put there (and I think a full example is a good idea)